Library/Homebrew/utils/lock.sh

```diff
@@ -0,0 +1,40 @@
+# The file descriptor used in the lock.
+# This FD should be used exclusively for lock purpose.
+# Any value except 0(stdin), 1(stdout) and 2(stderr) can do the job.
+# It is recommended to choose a large number to avoid conflicting with
+# other FD opened by the script.
+LOCK_FD=200
```

Comment:
Can we make this a local variable and move it into `lock`? It could be passed to `_create_lock` via function arguments.